pkg/cmd/server/origin/master.go

```diff
@@ -410,7 +410,10 @@ func (c *MasterConfig) GetRestStorage() map[string]rest.Storage {
 	checkStorageErr(err)
 	buildConfigRegistry := buildconfigregistry.NewRegistry(buildConfigStorage)
 
-	deployConfigStorage, deployConfigStatusStorage, deployConfigScaleStorage, err := deployconfigetcd.NewREST(c.RESTOptionsGetter, c.DeploymentConfigScaleClient())
+	oc, kc := c.DeploymentConfigInstantiateClients()
```

Comment:
not `oc, kc` here.  These are specifically for deploymentconfiginstantiate.  If we didn't have a cyclical dependency problem we'd use service accounts.